numba/cuda/tests/cudapy/test_constmem.py

```diff
@@ -134,7 +134,7 @@ def test_const_array_3d(self):
         self.assertTrue(np.all(A == CONST3D))
 
         if not ENABLE_CUDASIM:
-            if cuda.runtime.get_version() in ((8, 0), (9, 0), (9, 1)):
+            if cuda.runtime.get_version() in ((8, 0), (9, 0), (9, 1), (11, 2)):
```

Comment:
What about the versions in between, e.g. 10? Are they all covered by `else`? Please remind me why :) (and then put in a note) thanks!

Reply:
CUDA 9.2 - 11.1 uses two `f32` loads to load the complex, and the other versions use a load of one `2x f32` vector to load the complex. I don't know why though!

Reply:
ah yes, that was it :) thanks! Perhaps this could be put in as a note?

Reply:
(will add a comment)

Reply:
thanks!